matdeeplearn/preprocessor/processor.py

```diff
@@ -41,13 +42,14 @@ def process_data(dataset_config):
         r=cutoff_radius,
         n_neighbors=n_neighbors,
         edge_steps=edge_steps,
+        transforms=dataset_config.get("transforms", []),
```

Comment:
See my other comment above about maybe making the default list to be this
```suggestion
        transforms=dataset_config.get("transforms", ["GetY"]),
```

Reply:
Same as above.